test/api/ouroborous.cpp

```diff
@@ -24,6 +24,7 @@
  ** below, in SMT-LIBv2 form (but they're good for all languages).
  **/
 
+#include <cassert>
```

Comment:
Why are we using raw `assert` here?

Reply:
Because this is a test that uses CVC4 but is not part of CVC4.